packages/lib/src/FormController/FormController.ts

```diff
@@ -151,7 +151,7 @@ export class FormController {
 
   // runs validation for particular field
   protected runFieldLevelValidation = (name: string) => {
-    return this.fieldValidations[name](utils.getValue(this.formattedValues, name), this.formattedValues);
+    return this.fieldValidations[name]?.(utils.getValue(this.formattedValues, name), this.formattedValues);
```

Comment:
Ok, I see. But in this case it's better to short-circuit entire logic behind validateField.
I'm talking about 
https://github.com/iaroslavshvets/formagus/blob/0bf1cb12bb5ca2e26c7fa1457f6ede00068f0ae1/packages/lib/src/FormController/FormController.ts#L412
First we can change it to get field on first line and in case there is no field level validation just return without even setting `this.setIsValidating(true);` 

WDYT?